### DISCUSSION TITLE
GHI to POA Transposition Odd Results

### DISCUSSION THREAD

**Discussion by whoami-anvil** [pvlib/pvlib-python#1485] [Q&A] [answered]
I am working on a project that requires GHI to POA Transposition and from the data that I have put in, I am getting weird drops in the predicted values in the afternoon. I am using the erbs model function to get DNI and DHI and I am using the get_total_irradiance function to transfer this data to POA. Is this an expected result? Attached below are my graphs of GHI and DHI vs Time, GHI and DNI vs Time, GHI, POA, and Predicted POA vs Time, and POA and Predicted POA vs Time. Thank you in advance!
<img width="1350" alt="Screen Shot 2022-07-01 at 10 04 20 AM" src="https://user-images.githubusercontent.com/77754146/176910931-9049ff4d-b89e-44a4-98e3-8f0e0c306fce.png">
<img width="1331" alt="Screen Shot 2022-07-01 at 10 04 34 AM" src="https://user-images.githubusercontent.com/77754146/176910934-a4674a51-5b51-4b06-9596-4ccd77589d9d.png">

**Comment by kandersolar** [comment, marked as answer]
Hi @whoami-anvil, no, these results are not expected for a correct model.  This kind of thing can happen because of time zone mismatch between the various datetime indexes in your data, for example perhaps your irradiance data are in local time and the calculated solar positions are in UTC, or vice versa.  The fact that your measured irradiance peaks around 17:00 instead of closer to 12:00 also suggests a time zone issue.  I suspect if you compare your measured irradiance data with the calculated solar zeniths curve you will find they are misaligned in time.

I suggest making sure your measured data are localized to the correct time zone (i.e. that `df.index.tz` is appropriate, assuming you're using pandas) and basing all calculations (solar position, etc) on the tz-localized index of the measured data to ensure consistency.  If you haven't worked with time zones much in python, this page in our docs might be helpful: https://pvlib-python.readthedocs.io/en/stable/user_guide/timetimezones.html

**Reply by whoami-anvil** [reply]
Thank you for your response! I am going to fiddle around with my code for a bit to see if that is the issue.

**Reply by whoami-anvil** [reply]
It seems like that was the issue! Thank you very much for you help!
